### DISCUSSION TITLE
Is Remotely safe for commercial usage?

### DISCUSSION THREAD

**Discussion by venzonite** [immense/Remotely#442] [General]
Hi, what's your opinion on the security of this app? I plan to use it commercially.
Have you noticed any vulnerabilities? Have you any suggestions how to improve security of Remotely?

**Comment by rcschaff82** [comment]
It's safe if you know what you are doing.  A couple things to secure your remotely.

1)  Always make SSL required through the frontend  ( I use nginx, which makes this easy)
2)  Use DDNS on your admin connections, and use a good firewall to block the frontend port except those DDNS entries (I use configserv firewall)

**Comment by netroti128** [comment]
Hi, can you share example configserv firewall configuration?
Thanks

**Reply by rcschaff82** [reply]
on Nginx, I use port 83 with SSL.

I leave port 83 closed on csf.   

Each remote computer and my computer all have dynamic dns clients installed.  Then add the hostnames of those into csf.dyndns

Any hostname/ip that bypasses the firewall has access to all ports on the server.

Side note: If you client machines have static IP's, you can add them to csf.allow
